.env.dist

```diff
@@ -148,4 +148,12 @@ MAILER_DSN=smtp://null
 
 # Tax split configuration for new product order & backlog product order, for new service order & backlog service order
 #ENABLE_PAYMENT_TAX_SPLIT=false
-#STRIPE_TAX_ACCOUNT=acct_xxxxxx
\ No newline at end of file
+#STRIPE_TAX_ACCOUNT=acct_xxxxxx
+
+# Define the additional metadata sent to Stripe account on create/update
+# Default to {"operator_internal_id": "operatorInternalId"}, can be used to add custom metadata to the Stripe account (multiple fields can be separated by comma)
+STRIPE_ACCOUNT_METADATA={"operator_internal_id": "operatorInternalId"}
+
+# Set the amount value as [total amount - taxes - shipping taxes] on transfer creation to Stripe (default set to true) when ENABLE_PAYMENT_TAX_SPLIT=false
+# If ENABLE_PAYMENT_TAX_SPLIT=true, the transfer amount is calculated with the same formula [total amount - taxes - shipping taxes]
+ENABLE_SUBSTRACT_TAXES_FROM_TRANSFER_AMOUNT=true
```

Comment:
You've got a typo in "subtract", if that's the word you're going for.

The file could use a newline at the end as well.